### PR TITLE
master-lps-102636

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-102636

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 1]
```diff
@@ -0,0 +1,59 @@
+definition {
+
+	macro configurationOpenGraph {
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: configureOpenGraph

configuration is not a verb

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		AssertElementPresent(
+			key_title = "Open Graph",
+			locator1 = "Header#H4_TITLE");
+
+		if (isSet(customTitle)) {
+			var checkboxName = "Use Custom Title";
+
+			if (IsNotChecked(locator1 = "Checkbox#ANY_CHECKBOX")) {
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: isNotChecked

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+				Click(locator1 = "Checkbox#ANY_CHECKBOX");
+
+				Type(
+					locator1 = "TextInput#OPEN_GRAPH_CUSTOM_TITLE",
+					value1 = "${customTitle}");
+			}
+			else {
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: Remove this statement. A user should not be able to directly type into the text box unless the checkbox is checked. That's a bug

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+			if (IsNotChecked(locator1 = "Checkbox#ANY_CHECKBOX")) {
+				Click(locator1 = "Checkbox#ANY_CHECKBOX");
+
+				Type(
+					key_text = "openGraphDescription",
+					locator1 = "TextArea#ANY",
+					value1 = "${customDescription}");
+			}
+			else {
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: Same here

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+				Type.sendKeys(
+					locator1 = "TextInput#OPEN_GRAPH_CUSTOM_TITLE",
+					value1 = "${customTitle}");
+			}
+		}
+
+		if (isSet(customDescription)) {
+			var checkboxName = "Use Custom Description";
+
+			if (IsNotChecked(locator1 = "Checkbox#ANY_CHECKBOX")) {
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: isNotChecked

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+					key_text = "openGraphDescription",
+					locator1 = "TextArea#ANY",
+					value1 = "${customDescription}");
+			}
+		}
+
+		PortletEntry.save();
+	}
+
+	macro viewNoOpenGraphProperty {
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: Properties

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+		for (var ogProperty : list "${ogPropertyList}" ) {
+			AssertHTMLSourceTextNotPresent(value1 = "${ogProperty}");
+		}
+	}
+
+	macro viewOpenGraphProperty {
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: Properties

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/OpenGraph.testcase [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+		}
+	}
+
+	@description = "This ensures that og tags will not appear when disable the opengraph."
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/OpenGraph.testcase]: This ensures that OG tags will not appear when Open Graph is disabled.

Please make sure to check your grammar and punctuation before you submit.

**Comment by austinchiang** [comment]
Please resend

**Pull request closed without merging**